src/modules/terms/terms.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiBadRequestResponse, ApiBody, ApiCreatedResponse, ApiInternalServerErrorResponse, ApiNotFoundResponse, ApiOperation } from '@nestjs/swagger';
+import { RESPONSE } from '@ukef/constants';
+
+import { CreateFacilityTermRequestItem } from './dto/create-facility-term-request.dto';
+import { TermsService } from './terms.service';
+
+@Controller('terms')
+export class TermsController {
+  constructor(private readonly service: TermsService) {}
+
+  @Post('facilities')
```

Comment:
It'd be great if we could write API tests for this endpoint too. Alex is still working out how to do these on his ticket so it'd make sense to do API tests for this endpoint as a separate ticket to avoid the overlap. I've raised APIM-461 for that

Reply:
(so there's nothing to do in this PR for it!)